packages/draft-js-import-markdown/src/__tests__/stateFromMarkdown-test.js

```diff
@@ -61,6 +61,47 @@ describe('stateFromMarkdown', () => {
       },
     ]);
   });
+  it('should correctly handle images with alt text', () => {
+    const src = 'https://google.com/logo.png';
+    const alt = 'The Google Logo';
+    let markdown = `![${alt}](${src})`;
+    let contentState = stateFromMarkdown(markdown, {
+      parserOptions: {atomicImages: true},
```

Comment:
@mxstbr: I added this here. Unless you have an objection to this approach, I'll go ahead and land this when the CI passes and I'll cut a release.

Reply:
Perfect :100: